demo-app/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,8 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionSha256Sum=544c35d6bd849ae8a5ed0bcea39ba677dc40f49df7d1835561582da2009b961d
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
+networkTimeout=10000
```

Comment:
Is this deliberately low so it can show errors more easily?

Reply:
I'm sure this is autogenerated boilerplate from running gradle wrapper.

Reply:
Ah. It's a demo app, so this is more good than bad. Hope no one takes this and uses it in production lol